### DISCUSSION TITLE
Can I get an old snapshot?

### DISCUSSION THREAD

**Discussion by yaoluoye** [filecoin-project/lotus#8523] [General Q&A]
I can download the newest snapshot; it is https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/complete_chain_with_finality_stateroots_latest.car.

How can get an old snapshot, such as https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_xxx_2022-03-01_xxx.car

**Comment by LexLuthr** [comment]
@yaoluoye What is the use case for an old snapshot?

**Reply by yaoluoye** [reply]
I wonder if the data is identical in the two scenes.
1. Download the latest snapshot on 03/01/2022, then start to sync; assume no error happened.
2. Download the latest snapshot today, then start to sync.

**Reply by LexLuthr** [reply]
It would be same. Otherwise, blockchain wouldn't work. Older snapshot will allow you to have older tipset on your local system to run historical queries against. That's the only use case where you need older snapshots or full nodes.

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new input from the author for more than 7 days.
